src/contracts/core/StrategyManager.sol

```diff
@@ -202,6 +206,17 @@ contract StrategyManager is
         return (isDeleted, existingWithdrawalRoot);
     }
 
+    /**
+     * If true for a strategy, a user cannot depositIntoStrategyWithSignature into that strategy for another staker
+     * and also when performing DelegationManager.queueWithdrawals, a staker can only withdraw to themselves.
+     * Defaulted to false for all existing strategies.
+     * @param strategy The strategy to set `creditTransfersDisabled` value to
+     * @param value bool value to set `creditTransfersDisabled` to
+     */
+    function setCreditTransfersDisabled(IStrategy strategy, bool value) external onlyOwner {
+        _setCreditTransfersDisabled(strategy, value);
+    }
```

Comment:
Should this be an `onlyOwner` method, or a `onlyStrategyWhitelister` method? I'm thinking the latter:

The strategy whitelister can set transfers disabled by removing an existing strategy, then re-adding it with transfers disabled.

And if the strategy isn't already whitelisted, they can of course add it with transfers disabled.

Reply:
Additional thing: this method should probably check that the strategy is whitelisted

Reply:
I agree this should probably be controlled by the `strategyWhitelister` role.

> this method should probably check that the strategy is whitelisted

I think I actually disagree with this. seems fine to allow setting this prior to whitelisting.

Reply:
> I think I actually disagree with this. seems fine to allow setting this prior to whitelisting.

ok fair, that would make the ops around whitelisting easier

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/392/commits/323fa4e387941179468d541a4c9d9470c2382bd1#diff-31bf779120d094208ed8d64498e5f0c894da92b024a34723b583bef49524a877R219
Updated to `onlyStrategyWhitelister`